Fix paths in sargraph script

--- sysstat-9.0.6.1.orig/contrib/sargraph/sargraph
+++ sysstat-9.0.6.1/contrib/sargraph/sargraph
@@ -5,12 +5,12 @@
 # Our dependencies
 ZENITY="/usr/bin/zenity"
 XSLTPROC="/usr/bin/xsltproc"
-SADF="/usr/local/bin/sadf"
+SADF="/usr/bin/sadf"
 GNUPLOT="/usr/bin/gnuplot"
 MKTEMP="/bin/mktemp"
 FIND="/usr/bin/find"
-SORT="/bin/sort"
-CUT="/bin/cut"
+SORT="/usr/bin/sort"
+CUT="/usr/bin/cut"
 
 # sar / sysstat DTD is published here:
 # http://pagesperso-orange.fr/sebastien.godard/sysstat.dtd
@@ -367,7 +367,7 @@
 
 # Prompt for sar file
 
-SARLIST=`$FIND /var/log/sa -name 'sa[0-9][0-9]' -printf '%T@,%p\n'| $SORT -n -r | $CUT -d, -f2`
+SARLIST=`$FIND /var/log/sysstat -name 'sa[0-9][0-9]' -printf '%T@,%p\n'| $SORT -n -r | $CUT -d, -f2`
 SARFILE=`$ZENITY --list --text "Select data source" --column "sar file" $SARLIST`
 if [ "$SARFILE" == "" ]; then
     exit
